### ISSUE TITLE
Hook logs are missing in terminal when there are nested Describe/Context blocks

### ISSUE THREAD

**Issue by ntermartirosyan** [archfz/cypress-terminal-report#268]
Given there's a test with nested Describe/Context blocks
When test fails in the hook there are no logs from cypress commands in the terminal
Code example:
```
describe("Describe 1", () => {
  before(() => {
    cy.log("Log 1")
      .then(() => {
        expect(false, "Expect 1").to.be.true;
      });
  });

  describe("Nested Describe", () => {
    it("it 1", () => {
      cy.log("Log 2");
    });
  });
});
```
Result in Terminal : 
![image](https://github.com/user-attachments/assets/4fdc6d22-3ea8-4c5c-b001-4cf92aad399f)

When there is no nested Describe/Context block then the logs can be seen in the terminal
```
describe("Describe 1", () => {
  before(() => {
    cy.log("Log 1")
      .then(() => {
        expect(false, "Expect 1").to.be.true;
      });
  });

  // describe("Nested Describe", () => {
    it("it 1", () => {
      cy.log("Log 2");
    });
  // });
});
```
![image](https://github.com/user-attachments/assets/56e26465-c817-42df-be33-834ae884f256)

**Comment by ntermartirosyan** [comment]
Tested on version: 7.0.4
Cypress version: 13.12.0
these are my configs
```
require("cypress-terminal-report/src/installLogsCollector")({
  enableExtendedCollector: true,
  xhr: {
    printHeaderData: true,
    printRequestData: true,
  }
});

require("cypress-terminal-report/src/installLogsPrinter")(on, {
  printLogsToConsole: "always",
  includeSuccessfulHookLogs: true,
});
```
let me know if additional info is needed.

**Comment by archfz** [comment]
Released in 7.1.0
